pdns/dnsdist.cc

```diff
@@ -2082,9 +2084,9 @@ try
   string optstring;
   for(;;) {
 #ifdef HAVE_LIBSODIUM
-    int c=getopt_long(argc, argv, "a:hcde:C:k:l:vp:g:u:V", longopts, &longindex);
+    int c=getopt_long(argc, argv, "a:hce:C:k:l:v:g:u:V", longopts, &longindex);
```

Comment:
`--verbose` doesn't take any argument so the `:` after the `v` should go (same thing below).

Reply:
Fixed.